Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM maven:3.6.3-jdk-13 as build_jar
+RUN yum install git -y
+WORKDIR /jar
+COPY pom.xml .
+RUN mvn dependency:go-offline
+COPY src/ /jar/src/
+RUN mvn package
+
+FROM adoptopenjdk/openjdk13:alpine-jre
+WORKDIR /artipie
+COPY --from=build_jar /jar/target/artipie-jar-with-dependencies.jar /artipie/artipie.jar
```

Comment:
@Sammers21 it can be `COPY --from=build_jar /jar/target/artipie-jar-with-dependencies.jar .`

Reply:
I also can be as is. It is explicit and clear

Reply:
@Sammers21 ok, but let's use single style in the `Dockerfile`. If you think explicit version is clearer, then remove `WORKDIR`s and use full paths everywhere.